<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QMainWindow" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPalette{background:#dcdcdc;}*{outline:0px;color:#000000;}

QMainWindow {
border:8px solid #00bcd4;
font-family: Arial 1px solid #2a2d30;;
font-style: normal;
font-size: 12pt;
font-weight: bold;
}

QWidget[form=&quot;true&quot;],QLabel[frameShape=&quot;1&quot;]{
border:1px solid #535353;
border-radius:0px;
}

.QFrame{
border:1px solid #535353;
border-radius:3px;
}

QWidget[form=&quot;true&quot;],QLabel[frameShape=&quot;1&quot;]{
border:1px solid #B2B6B9;
border-radius:0px;
}

QWidget#widget_top&gt;QToolButton,QWidget#widget_top&gt;QLabel{
border-style:none;
border-radius:0px;
padding:5px;
color:#000000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QWidget#widget_top&gt;QToolButton:pressed,QWidget#widget_top&gt;QToolButton:hover,QWidget#widget_top&gt;QToolButton:checked,QWidget#widget_top&gt;QLabel:hover{
border-style:solid;
border-width:0px 0px 2px 0px;
border-color:#00BB9E;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

QGroupBox#gboxVideo&gt;QLabel,QFrame#gboxVideo&gt;QLabel{
color:#000000;
border:1px solid #A9A9A9;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QGroupBox#gboxVideo&gt;QLabel:focus,QFrame#gboxVideo&gt;QLabel:focus{
border:1px solid #FF0000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

QLineEdit,QTextEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox,QComboBox,QDateEdit,QTimeEdit,QDateTimeEdit{
border:1px solid #A9A9A9;
border-radius:3px;
padding:2px;
background:none;
selection-background-color:#E4E4E4;
selection-color:#000000;
}

QLineEdit:focus,QTextEdit:focus,QPlainTextEdit:focus,QSpinBox:focus,QDoubleSpinBox:focus,QComboBox:focus,QDateEdit:focus,QTimeEdit:focus,QDateTimeEdit:focus,QLineEdit:hover,QTextEdit:hover,QPlainTextEdit:hover,QSpinBox:hover,QDoubleSpinBox:hover,QComboBox:hover,QDateEdit:hover,QTimeEdit:hover,QDateTimeEdit:hover{
border:1px solid #A9A9A9;
}

QLineEdit[echoMode=&quot;2&quot;]{
lineedit-password-character:9679;
}

.QGroupBox{
border:1px solid #A9A9A9;
border-radius:5px;
margin-top:3ex;
}

.QGroupBox::title{
subcontrol-origin:margin;
position:relative;
left:10px;
}

.QPushButton,.QToolButton{
border-style:none;
border:1px solid #A9A9A9;
color:#000000;
padding:2px;
min-height:15px;
border-radius:5px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

.QPushButton:hover,QToolButton:hover{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

.QPushButton:pressed,QToolButton:pressed{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QToolButton#btnMenu,QPushButton#btnMenu_Min,QPushButton#btnMenu_Max,QPushButton#btnMenu_Close{
border-radius:3px;
color:#000000;
padding:3px;
margin:0px;
background:none;
border-style:none;
}

QToolButton#btnMenu:hover,QPushButton#btnMenu_Min:hover,QPushButton#btnMenu_Max:hover{
color:#FFFFFF;
margin:1px 1px 2px 1px;
background-color:rgba(51,127,209,230);
}

QPushButton#btnMenu_Close:hover{
color:#FFFFFF;
margin:1px 1px 2px 1px;
background-color:rgba(238,0,0,128);
}

QWidget#widget_left&gt;QPushButton,QWidget#widget_left&gt;QToolButton{
border-radius:0px;
color:#000000;
background:none;
border-style:none;
}

QWidget#widget_left&gt;QPushButton:hover,QWidget#widget_left&gt;QToolButton:hover{
color:#FFFFFF;
background-color:#00BB9E;
}

QWidget#widget_left&gt;QPushButton:checked,QWidget#widget_left&gt;QToolButton:checked,QWidget#widget_left&gt;QPushButton:pressed,QWidget#widget_left&gt;QToolButton:pressed{
color:#000000;
border-style:solid;
border-width:0px 0px 0px 2px;
border-color:#00BB9E;
background-color:#F0F0F0;
}

QRadioButton::indicator{
width:15px;
height:15px;
}

QRadioButton::indicator::unchecked{
image:url(:/themes/themes/icons/radiobutton_unchecked.png);
}

QRadioButton::indicator::unchecked:disabled{
image:url(:/themes/themes/icons/radiobutton_unchecked_disable.png);
}

QRadioButton::indicator::checked{
image:url(:/themes/themes/icons/radiobutton_checked.png);
}

QRadioButton::indicator::checked:disabled{
image:url(:/themes/themes/icons/radiobutton_checked_disable.png);
}

QGroupBox::indicator,QTreeWidget::indicator,QListWidget::indicator{
padding:0px -3px 0px 3px;
}

QCheckBox::indicator,QGroupBox::indicator,QTreeWidget::indicator,QListWidget::indicator{
width:13px;
height:13px;
}

QCheckBox::indicator:unchecked,QGroupBox::indicator:unchecked,QTreeWidget::indicator:unchecked,QListWidget::indicator:unchecked{
image:url(:/themes/themes/icons/checkbox_unchecked.png);
}

QCheckBox::indicator:unchecked:disabled,QGroupBox::indicator:unchecked:disabled,QTreeWidget::indicator:unchecked:disabled,QListWidget::indicator:disabled{
image:url(:/themes/themes/icons/checkbox_unchecked_disable.png);
}

QCheckBox::indicator:checked,QGroupBox::indicator:checked,QTreeWidget::indicator:checked,QListWidget::indicator:checked{
image:url(:/themes/themes/icons/checkbox_checked.png);
}

QCheckBox::indicator:checked:disabled,QGroupBox::indicator:checked:disabled,QTreeWidget::indicator:checked:disabled,QListWidget::indicator:checked:disabled{
image:url(:/themes/themes/icons/checkbox_checked_disable.png);
}

QCheckBox::indicator:indeterminate,QGroupBox::indicator:indeterminate,QTreeWidget::indicator:indeterminate,QListWidget::indicator:indeterminate{
image:url(:/themes/themes/icons/checkbox_parcial.png);
}

QCheckBox::indicator:indeterminate:disabled,QGroupBox::indicator:indeterminate:disabled,QTreeWidget::indicator:indeterminate:disabled,QListWidget::indicator:indeterminate:disabled{
image:url(:/themes/themes/icons/checkbox_parcial_disable.png);
}

QTimeEdit::up-button,QDateEdit::up-button,QDateTimeEdit::up-button,QDoubleSpinBox::up-button,QSpinBox::up-button{
image:url(:/themes/themes/icons/add_top.png);
width:10px;
height:10px;
padding:2px 5px 0px 0px;
}

QTimeEdit::down-button,QDateEdit::down-button,QDateTimeEdit::down-button,QDoubleSpinBox::down-button,QSpinBox::down-button{
image:url(:/themes/themes/icons/add_bottom.png);
width:10px;
height:10px;
padding:0px 5px 2px 0px;
}

QTimeEdit::up-button:pressed,QDateEdit::up-button:pressed,QDateTimeEdit::up-button:pressed,QDoubleSpinBox::up-button:pressed,QSpinBox::up-button:pressed{
top:-2px;
}

QTimeEdit::down-button:pressed,QDateEdit::down-button:pressed,QDateTimeEdit::down-button:pressed,QDoubleSpinBox::down-button:pressed,QSpinBox::down-button:pressed,QSpinBox::down-button:pressed{
bottom:-2px;
}

QComboBox::down-arrow,QDateEdit[calendarPopup=&quot;true&quot;]::down-arrow,QTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow,QDateTimeEdit[calendarPopup=&quot;true&quot;]::down-arrow{
image:url(:/themes/themes/icons/add_bottom.png);
width:10px;
height:10px;
right:2px;
}

QComboBox::drop-down,QDateEdit::drop-down,QTimeEdit::drop-down,QDateTimeEdit::drop-down{
subcontrol-origin:padding;
subcontrol-position:top right;
width:15px;
border-left-width:0px;
border-left-style:solid;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
border-left-color:#A9A9A9;
}

QComboBox::drop-down:on{
top:1px;
}

QMenuBar::item{
color:#000000;
background-color:#E4E4E4;
margin:0px;
padding:3px 10px;
}

QMenu,QMenuBar,QMenu:disabled,QMenuBar:disabled{
color:#000000;
background-color:#E4E4E4;
border:1px solid #A9A9A9;
margin:0px;
}

QMenu::item{
padding:3px 20px;
}

QMenu::indicator{
width:13px;
height:13px;
}

QMenu::item:selected,QMenuBar::item:selected{
color:#000000;
border:0px solid #A9A9A9;
background:#DBDBDB;
}

QMenu::separator{
height:1px;
background:#A9A9A9;
}

QProgressBar{
min-height:10px;
background:#E4E4E4;
border-radius:5px;
text-align:center;
border:1px solid #E4E4E4;
}

QProgressBar:chunk{
border-radius:5px;
background-color:#A9A9A9;
}

QSlider::groove:horizontal{
background:#E4E4E4;
height:8px;
border-radius:4px;
}

QSlider::add-page:horizontal{
background:#E4E4E4;
height:8px;
border-radius:4px;
}

QSlider::sub-page:horizontal{
background:#A9A9A9;
height:8px;
border-radius:4px;
}

QSlider::handle:horizontal{
width:13px;
margin-top:-3px;
margin-bottom:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #F0F0F0,stop:0.8 #A9A9A9);
}

QSlider::groove:vertical{
width:8px;
border-radius:4px;
background:#E4E4E4;
}

QSlider::add-page:vertical{
width:8px;
border-radius:4px;
background:#E4E4E4;
}

QSlider::sub-page:vertical{
width:8px;
border-radius:4px;
background:#A9A9A9;
}

QSlider::handle:vertical{
height:14px;
margin-left:-3px;
margin-right:-3px;
border-radius:6px;
background:qradialgradient(spread:pad,cx:0.5,cy:0.5,radius:0.5,fx:0.5,fy:0.5,stop:0.6 #F0F0F0,stop:0.8 #A9A9A9);
}

QScrollBar:horizontal{
background:#E4E4E4;
padding:0px;
border-radius:6px;
max-height:12px;
}

QScrollBar::handle:horizontal{
background:#C1C1C1;
min-width:50px;
border-radius:6px;
}

QScrollBar::handle:horizontal:hover{
background:#A9A9A9;
}

QScrollBar::handle:horizontal:pressed{
background:#A9A9A9;
}

QScrollBar::add-page:horizontal{
background:none;
}

QScrollBar::sub-page:horizontal{
background:none;
}

QScrollBar::add-line:horizontal{
background:none;
}

QScrollBar::sub-line:horizontal{
background:none;
}

QScrollBar:vertical{
background:#E4E4E4;
padding:0px;
border-radius:6px;
max-width:12px;
}

QScrollBar::handle:vertical{
background:#C1C1C1;
min-height:50px;
border-radius:6px;
}

QScrollBar::handle:vertical:hover{
background:#A9A9A9;
}

QScrollBar::handle:vertical:pressed{
background:#A9A9A9;
}

QScrollBar::add-page:vertical{
background:none;
}

QScrollBar::sub-page:vertical{
background:none;
}

QScrollBar::add-line:vertical{
background:none;
}

QScrollBar::sub-line:vertical{
background:none;
}

QScrollArea{
border:0px;
}

QTreeView,QListView,QTableView,QTabWidget::pane{
border:1px solid #A9A9A9;
selection-background-color:#DBDBDB;
selection-color:#000000;
alternate-background-color:#C1C1C1;
gridline-color:#A9A9A9;
}

QTreeView::branch:closed:has-children{
margin:4px;
border-image:url(:/themes/themes/icons/branch_open.png);
}

QTreeView::branch:open:has-children{
margin:4px;
border-image:url(:/themes/themes/icons/branch_close.png);
}

QTreeView,QListView,QTableView,QSplitter::handle,QTreeView::branch{
background:#F0F0F0;
}

QTableView::item:selected,QListView::item:selected,QTreeView::item:selected{
color:#000000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QTableView::item:hover,QListView::item:hover,QTreeView::item:hover{
color:#000000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

QTableView::item,QListView::item,QTreeView::item{
padding:1px;
margin:0px;
}

QHeaderView::section,QTableCornerButton:section{
padding:3px;
margin:0px;
color:#000000;
border:1px solid #A9A9A9;
border-left-width:0px;
border-right-width:1px;
border-top-width:0px;
border-bottom-width:1px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

QTabBar::tab{
border:1px solid #A9A9A9;
color:#000000;
margin-right:0px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

QTabBar::tab:selected,QTabBar::tab:hover{
border-style:solid;
border-color:#00BB9E;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QTabBar::tab:top,QTabBar::tab:bottom{
padding:3px 8px 3px 8px;
}

QTabBar::tab:left,QTabBar::tab:right{
padding:8px 3px 8px 3px;
}

QTabBar::tab:top:selected,QTabBar::tab:top:hover{
border-width:2px 0px 0px 0px;
}

QTabBar::tab:right:selected,QTabBar::tab:right:hover{
border-width:0px 0px 0px 2px;
}

QTabBar::tab:bottom:selected,QTabBar::tab:bottom:hover{
border-width:0px 0px 2px 0px;
}

QTabBar::tab:left:selected,QTabBar::tab:left:hover{
border-width:0px 2px 0px 0px;
}

QTabBar::tab:first:top:selected,QTabBar::tab:first:top:hover,QTabBar::tab:first:bottom:selected,QTabBar::tab:first:bottom:hover{
border-left-width:1px;
border-left-color:#A9A9A9;
}

QTabBar::tab:first:left:selected,QTabBar::tab:first:left:hover,QTabBar::tab:first:right:selected,QTabBar::tab:first:right:hover{
border-top-width:1px;
border-top-color:#A9A9A9;
}

QTabBar::tab:last:top:selected,QTabBar::tab:last:top:hover,QTabBar::tab:last:bottom:selected,QTabBar::tab:last:bottom:hover{
border-right-width:1px;
border-right-color:#A9A9A9;
}

QTabBar::tab:last:left:selected,QTabBar::tab:last:left:hover,QTabBar::tab:last:right:selected,QTabBar::tab:last:right:hover{
border-bottom-width:1px;
border-bottom-color:#A9A9A9;
}

QStatusBar::item{
border:0px solid #E4E4E4;
border-radius:3px;
}

QToolBox::tab,QGroupBox#gboxDevicePanel,QGroupBox#gboxDeviceTitle,QFrame#gboxDevicePanel,QFrame#gboxDeviceTitle{
padding:3px;
border-radius:5px;
color:#000000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QToolTip{
border:0px solid #000000;
padding:1px;
color:#000000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QToolBox::tab:selected{
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #DBDBDB,stop:1 #C1C1C1);
}

QPrintPreviewDialog QToolButton{
border:0px solid #000000;
border-radius:0px;
margin:0px;
padding:3px;
background:none;
}

QColorDialog QPushButton,QFileDialog QPushButton{
min-width:80px;
}

QToolButton#qt_calendar_prevmonth{
icon-size:0px;
min-width:20px;
image:url(:/themes/themes/icons/calendar_prevmonth.png);
}

QToolButton#qt_calendar_nextmonth{
icon-size:0px;
min-width:20px;
image:url(:/themes/themes/icons/calendar_nextmonth.png);
}

QToolButton#qt_calendar_prevmonth,QToolButton#qt_calendar_nextmonth,QToolButton#qt_calendar_monthbutton,QToolButton#qt_calendar_yearbutton{
border:0px solid #000000;
border-radius:3px;
margin:3px 3px 3px 3px;
padding:3px;
background:none;
}

QToolButton#qt_calendar_prevmonth:hover,QToolButton#qt_calendar_nextmonth:hover,QToolButton#qt_calendar_monthbutton:hover,QToolButton#qt_calendar_yearbutton:hover,QToolButton#qt_calendar_prevmonth:pressed,QToolButton#qt_calendar_nextmonth:pressed,QToolButton#qt_calendar_monthbutton:pressed,QToolButton#qt_calendar_yearbutton:pressed{
border:1px solid #A9A9A9;
}

QCalendarWidget QSpinBox#qt_calendar_yearedit{
margin:2px;
}

QCalendarWidget QToolButton::menu-indicator{
image:None;
}

QCalendarWidget QTableView{
border-width:0px;
}

QCalendarWidget QWidget#qt_calendar_navigationbar{
border:1px solid #A9A9A9;
border-width:1px 1px 0px 1px;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #E4E4E4,stop:1 #A2A2A2);
}

QComboBox QAbstractItemView::item{
min-height:20px;
min-width:10px;
}

QTableView[model=&quot;true&quot;]::item{
padding:0px;
margin:0px;
}

QTableView QLineEdit,QTableView QComboBox,QTableView QSpinBox,QTableView QDoubleSpinBox,QTableView QDateEdit,QTableView QTimeEdit,QTableView QDateTimeEdit{
border-width:0px;
border-radius:0px;
}

QTableView QLineEdit:focus,QTableView QComboBox:focus,QTableView QSpinBox:focus,QTableView QDoubleSpinBox:focus,QTableView QDateEdit:focus,QTableView QTimeEdit:focus,QTableView QDateTimeEdit:focus{
border-width:0px;
border-radius:0px;
}

QLineEdit,QTextEdit,QPlainTextEdit,QSpinBox,QDoubleSpinBox,QComboBox,QDateEdit,QTimeEdit,QDateTimeEdit{
background:#F0F0F0;
}

*:disabled{
background:#F0F0F0;
border-color:#E4E4E4;
}

/*TextColor:#000000*/
/*PanelColor:#F0F0F0*/
/*BorderColor:#A9A9A9*/
/*NormalColorStart:#E4E4E4*/
/*NormalColorEnd:#A2A2A2*/
/*DarkColorStart:#DBDBDB*/
/*DarkColorEnd:#C1C1C1*/
/*HighColor:#00BB9E*/
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widgetMain" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>15</y>
      <width>541</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>11</y>
       <width>521</width>
       <height>341</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>531</width>
      <height>67</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="ClickableLabel" name="lb_mrx">
         <property name="text">
          <string>Mofadal Altyeb</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="pbar">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QProgressBar {
border: 1px solid black;
text-align: top;
padding: 1px;
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #fff,
stop: 0.4999 #eee,
stop: 0.5 #ddd,
stop: 1 #eee );
width: 15px;
}

QProgressBar::chunk {
background: QLinearGradient( x1: 0, y1: 0, x2: 1, y2: 0,
stop: 0 #78d,
stop: 0.4999 #46a,
stop: 0.5 #45a,
stop: 1 #238 );
border-bottom-right-radius: 7px;
border-bottom-left-radius: 7px;
border: 1px solid black;
}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_stop">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/qss_icons/rc/window_close_disabled@2x.png</normaloff>:/qss_icons/rc/window_close_disabled@2x.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_screen_shot">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>Screen Shot</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btn_disable_sec_boot">
         <property name="styleSheet">
          <string notr="true">QPalette{background:#a9a9a9;}*{outline:0px;color:#000000;}

QMainWindow {
border:3px solid #005fa3;
font-family: Arial 1px solid #a9a9a9;
font-style: normal;
font-size: 12pt;
font-weight: bold;
}

.QFrame {
border:1px solid #a9a9a9;
border-radius:3px;
}

QWidget[form=&quot;true&quot;],QLabel[frameShape=&quot;1&quot;]{
border:1px solid #a9a9a9;
border-radius:0px;
}

QWidget#widget_top&gt;QToolButton,QWidget#widget_top&gt;QLabel{
border-style:none;
border-radius:0px;
padding:5px;
color:#000000;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #a9a9a9,stop:1 #a9a9a9);
}

QWidget#widget_top&gt;QToolButton:pressed,QWidget#widget_top&gt;QToolButton:hover,QWidget#widget_top&gt;QToolButton:checked,QWidget#widget_top&gt;QLabel:hover{
border-style:solid;
border-width:0px 0px 2px 0px;
border-color:#a9a9a9;
background:qlineargradient(spread:pad,x1:0,y1:0,x2:0,y2:1,stop:0 #a9a9a9,stop:1 #a9a9a9);
}
</string>
         </property>
         <property name="text">
          <string>Disable Auth</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_dump_pl">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Read Preloader</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_crash_pl">
         <property name="text">
          <string>Crash PL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_vivo_demo">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Vivo Demo Remove</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_reboot_meta">
         <property name="text">
          <string>Reboot META MODE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>567</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>:/images/24px.png</normaloff>:/images/24px.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset>
     <normaloff>:/images/open482.png</normaloff>:/images/open482.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="icon">
    <iconset>
     <normaloff>:/images/save32px.png</normaloff>:/images/save32px.png</iconset>
   </property>
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset>
     <normaloff>:/images/excel48.png</normaloff>:/images/excel48.png</iconset>
   </property>
   <property name="text">
    <string>Export Excel</string>
   </property>
   <property name="shortcut">
    <string>Alt+E</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/images/print48px.png</normaloff>:/images/print48px.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="icon">
    <iconset>
     <normaloff>:/images/connect_creating.png</normaloff>:/images/connect_creating.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normaloff>:/images/stop64.png</normaloff>:/images/stop64.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDevice_Set">
   <property name="icon">
    <iconset>
     <normaloff>:/images/setting48.png</normaloff>:/images/setting48.png</iconset>
   </property>
   <property name="text">
    <string>Device Set</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>:/images/about48px.png</normaloff>:/images/about48px.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset>
     <normaloff>:/images/start48.png</normaloff>:/images/start48.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Alt+R</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ClickableLabel</class>
   <extends>QLabel</extends>
   <header location="global">clickablelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
